src/main/clojure/clara/rules/engine.cljc

```diff
@@ -524,7 +524,7 @@
 ;; Record representing alpha nodes in the Rete network,
 ;; each of which evaluates a single condition and
 ;; propagates matches to its children.
-(defrecord AlphaNode [id env children activation]
+(defrecord AlphaNode [id env children activation type]
```

Comment:
So the id part of #237 has already been done?

Also, should it perhaps be `fact-type` to be clearer?

Reply:
yes, and i have updated it to be fact-type